src/spikeinterface/sorters/runsorter.py

```diff
@@ -314,7 +320,7 @@ def stop(self):
     def run_command(self, command):
         if self.mode == "docker":
             res = self.docker_container.exec_run(command)
-            return str(res.output)
+            return res.output.decode("utf-8")
```

Comment:
I would do res.output.decode("utf-8", errors="ignore") to be safer no ?

Reply:
done!